sky/clouds/gcp.py

```diff
@@ -347,6 +358,46 @@ def check_credentials(self) -> Tuple[bool, Optional[str]]:
                 'For more info: '
                 'https://skypilot.readthedocs.io/en/latest/getting-started/installation.html'  # pylint: disable=line-too-long
             )
+
+        # Check APIs.
+        project_id = self.get_project_id()
+        apis = (
+            ('compute', 'Compute Engine'),
+            ('cloudresourcemanager', 'Cloud Resource Manager'),
+            ('iam', 'Identity and Access Management (IAM)'),
```

Comment:
Should we also enable TPU API for users?
https://console.cloud.google.com/apis/library/tpu.googleapis.com?_ga=2.67630993.1652250969.1665079665-1061309080.1630272145

Reply:
Ah thanks for verifying this!